src/main/scala/org/scalameter/Parameter.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+case class Parameter[T: Pickler](key: String) extends PicklerBasedKey[T] {
```

Comment:
Same here - instead of making picklers serializable and embedding them into `Parameter` objects, embed their class names.

This way, it is clear from the class definition what the members are - in the current implementation, the `Parameter` class "secretly" gets the `Pickler` field.